docs/topics/writing.rst

```diff
@@ -1,8 +1,7 @@
 Writing Datasets
 =================
 
-.. todo::
-
+..
```

Comment:
```suggestion
..
    TODO:

```

Reply:
@adamjstewart I didn't want to lose the "todo" comments. Let's either have TODO in there without the plugin or keep the todo directive and add the plugin. I think I prefer your original idea after all. Sorry about the flip-flop.

Reply:
Okay, kept the todo directive and added the plugin.